hack/check-commit-message.sh

```diff
@@ -4,11 +4,12 @@ set -o nounset
 
 commit_file=${1}
 commit_message="$(cat ${commit_file})"
-valid_commit_regex='^(NO-ISSUE|Merge|Revert|((Bug |[A-Z]+-|#)[0-9]+:))'
+valid_commit_regex='^(NO-ISSUE|Merge|Revert|([A-Z]+-[0-9]+:))'
+valid_github_regex='^((Fixes|fixes|Closes|closes): #[0-9]+)'
```

Comment:
Please note here you require a ' ' after :

Reply:
I would also accept Issue #xx

because it can be related to an issue, but neither fixes it or closes it.

Reply:
Done